packages/webcord-deb/webcord-deb.pacscript

```diff
@@ -1,9 +1,9 @@
 name="webcord-deb"
 gives="webcord"
-version="3.10.1"
+version="4.0.0"
 replace="${gives}"
 url="https://github.com/SpacingBat3/WebCord/releases/download/v${version}/webcord_${version}_amd64.deb"
 repology=("project: webcord")
 description="A Discord and Fosscord client implemented directly without Discord API."
-hash="78901c1f0eecee91c7922eb163540ef136b6779b0f1eb10c54676a4013b3c9f3"
+hash="0e5f2070a060f1e10b0a427a11347ce825b113428ab1ad120d76cbe73a3b92b6"
 maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
```

Comment:
Why? This is not `g4music`. Remove it.

Reply:
Remove your last commit, don't make a new commit to remove it.